resources/hooks/local/commit-msg

```diff
@@ -1,5 +1,11 @@
 #!/bin/sh
 
+if ! command -v $(EXEC_GRUMPHP_COMMAND) &> /dev/null
```

Comment:
The exec command can also be a command with cli options and attributes.
For example:

```
grumphp:
    git_hook_variables:
        EXEC_GRUMPHP_COMMAND: '/usr/local/bin/php72'
        EXEC_GRUMPHP_COMMAND: 'lando php'
        EXEC_GRUMPHP_COMMAND: 'fin exec php'
        EXEC_GRUMPHP_COMMAND: ['php', '-c /custom/config.ini']
        EXEC_GRUMPHP_COMMAND: ['docker-compose', 'run', '-T', '--rm', '--no-deps', 'php']
        EXEC_GRUMPHP_COMMAND: 'docker run --rm -it -v $(pwd):/grumphp -w /grumphp webdevops/php:alpine'
        EXEC_GRUMPHP_COMMAND: 'ddev exec php'
```

This won't work in that situation